src/main.ts

```diff
@@ -13,7 +13,7 @@ setInvite('939613684592934992', '16384');
 setup(new URL('../src/.env', import.meta.url));
 
 await load(new URL('../src/locales', import.meta.url));
-await init();
+await init({ fallbackLng: 'en-US' });
```

Comment:
```suggestion
await init({
	lng: 'en-US',
	fallbackLng: 'en-US',
	load: 'all',
	returnEmptyString: false,
	returnNull: false
});
```

Imagine comparing with Skyra smh

Reply:
> `lng`: language to use (overrides language detection). Default: `undefined`

Non-applicable, we always used fixedT.

> `load`: strategy to define which language codes to lookup. Default: `'all'`

Already default.

> `returnEmptyString`: allows empty string as valid translation. Default: `true`

Yeah, needs to be set.

> `returnNull`: allows null values as valid translation. Default: `true`

Also needs to be set.

---


I'll add `returnEmptyString` and `returnNull`, but not the other two.